scapy/contrib/automotive/someip.py

```diff
@@ -213,6 +220,13 @@ def fragment(self, fragsize=1392):
 
         return lst
 
+    def guess_payload_class(self, payload):
+        """Defines if next payload is SD layer or default layer"""
+        if self.getfieldval("msg_id").srv_id == 0xffff:
```

Comment:
Can you use `self.msg_id.srv_id` instead?

Reply:
Yes, sure